test/e2e/service_cleanup.py

```diff
@@ -60,10 +60,17 @@ def delete_snapshot(snapshot_name: str):
         logging.info(f"Deleted replication group {snapshot['ReplicationGroupId']}")
 
 
+<<<<<<< HEAD
 def delete_non_default_user(user_id: str):
     ec = boto3.client("elasticache")
     ec.delete_user(UserId=user_id)
     logging.info(f"Deleted non default user {user_id}")
+=======
+def delete_log_group(log_group_name: str):
+    logs = boto3.client("logs")
+    logs.delete_log_group(logGroupName=log_group_name)
+    logging.info(f"Deleted CW log group {log_group_name}")
+>>>>>>> Add LogDelivery to RG Coverage
 
```

Comment:
I think this needs resolving git conflicts

Reply:
Ah :( Thanks!